shoop/simple_cms/templates/shoop/simple_cms/page.jinja

```diff
@@ -6,9 +6,21 @@
 {% block breadcrumb %}
     <ol class="breadcrumb">
         <li><a href="/"><i class="glyphicon glyphicon-home"></i></a></li>
+        {% for ancestor in page.get_ancestors() if ancestor.is_visible() %}
+            <li><a href="/{{ ancestor.url }}">{{ ancestor.title }}</a></li>
+        {% endfor %}
         <li class="active">{{ page.title }}</li>
     </ol>
 {% endblock %}
 {% block content %}
     {{ page.get_html()|safe }}
+    {% if page.list_children_on_page %}
+        {% for child in page.children.all().order_by("-available_from")  %}
+            <div class="row">
+                <a href="/{{ child.url }}"><h2>{{ child.title }}</h2></a>
+                <small>{{ child.available_from|datetime }}</small>
```

Comment:
Yes. Since the task is talking about articles and such I think it's important to show published date in the pages list.